packages/ui-extensions-server-kit/src/styles/theme.module.scss

```
@@ -188,6 +189,10 @@
   --x-icon-size-large: 40px;
 
   --p-font: 'SF Mono', monospace;
+  // custom colors
+  --p-critical:hsl(11, 100%, 43%);
```

Sorry haha tiny typo
```suggestion
  --p-critical: hsl(11, 100%, 43%);
```